routes/pointofinterest/route.js

```diff
@@ -14,7 +14,7 @@ router.get('/sample', (req, res) => {
     if (err) {
```

Comment:
It is not in this PR bu you should avoid SELECT * in your SQL request (l. 14)

Reply:
done